### PR TITLE
Update upgrade.md

### PR DESCRIPTION
**Pull request by lfbrock**
Update instructions for upgrading TE to EE to include saving a setting to force a refresh, (this is a temporary work around until https://mattermost.atlassian.net/browse/PLT-4190 is completed)

### CODE REVIEW

**Comment by jasonblais** [comment]
+1

File: source/administration/upgrade.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ If your Mattermost server has duplicate accounts (users with multiple accounts i
 5. Reset the Mattermost account as the directory owner by typing:
  - `sudo chown -R mattermost:mattermost <path-to-your-mattermost-folder>`
  - `sudo chmod -R g+w <path-to-your-mattermost-folder>`
+6. To force the client to refresh for your users, save a setting in the System Console. For example, go to **System Console** > **General** > **Configuration**, and copy the Listen Address. Then paste the Listen Address back into the field, and press **Save**.
```
Comment (it33) [inline source/administration/upgrade.md]: Propose `To force the client to refresh for your users...` > `To ensure clients applications don't encounter issues with out-of-sync caches, you can force clients to refresh after a server upgrade by changing any setting in the System Console.`

Also, is there a ticket to improve this functionality in future so the forced refresh automatically happens after a server upgrade?

**Comment by it33** [comment]
Sorry, just noticed the tracking ticket is there for the full fix, please ignore that part of the feedback.

File: source/administration/upgrade.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ To start, select one of the following guides:
       1. Run `platform -version` to check the current version of your Mattermost server
       2. Determine the appropriate next upgrade for your server:
           - Mattermost `v3.0.x` and above can upgrade directly to Mattermost `v3.4.x`
-              - Note: Upgrading to `v3.4.x` will cause existing public links to break. 
+              - Note: If public links are enabled, upgrading to `v3.4.x` will invalidate existing public links due to a security upgrade allowing admins to invalidate links by resetting a public link salt from the System Console.
```
Comment (it33) [inline source/administration/upgrade.md]: Updating note on 3.4 breaking public links to explain why they're breaking.

File: source/administration/upgrade.md (hunk 1 above)
Comment (lfbrock) [inline source/administration/upgrade.md]: Updated the text.

File: source/administration/upgrade.md (hunk 2 above)
Comment (yangchen1) [inline source/administration/upgrade.md]: +!

**Comment by yangchen1** [comment]
+1

**Pull request merged**